custom-scripts/simple.sh

```diff
@@ -1,3 +1,8 @@
 #!/bin/bash
 
-echo "This is a custom script!"
+if  [ "$IS_MASTER" = "1" ]; then
+    echo "This is a custom script running on just the master!"
+fi
+
+echo "This is a custom script running all workers and the master!"
```

Comment:
Maybe add that we currently always deploy a worker on the master node, so we do not support a 'worker only' install.

Reply:
Nm... just saw docs below.